core/components/header/cart-icon.tsx

```diff
@@ -40,12 +41,12 @@ export const CartIcon = ({ count: serverCount }: CartIconProps) => {
   }, [serverCount, locale, setCount]);
 
   if (!count) {
-    return <ShoppingCart aria-label="cart" />;
+    return <ShoppingCart aria-label={t('cart')} />;
```

Comment:
I know you were just updating the current strings, but going off the rules of aria we should only use `aria-` roles if it's absolutely necessary. We should be using the `<title>` element for SVGs instead:
```jsx
<ShoppingCart>
  <title>{t('cart')}</title>
</ShoppingCart>
```